src/types/Ctype.ts

```diff
@@ -25,20 +33,19 @@ export interface IClaimInput {
   title: string
   description?: string
   type: string
+  owner?: string
 }
 
 export class CType implements ICType {
-  public static fromObject(obj: ICType): CType {
+  public static fromObject(obj: CTypeMetadata): CType {
     const newCtype = Object.create(CType.prototype)
     return Object.assign(newCtype, obj)
   }
 
-  public metaData: {
-    author: string
-  }
   public readonly cType: sdk.ICType
-
+  public metadata: IMetadata
```

Comment:
You might be able to use `ICTypeMetadata['metadata'] ` to shorten your import

Reply:
Yes